src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -283,6 +283,22 @@ type Object struct {
 
 	// Prealloc enables memory preallocation
 	Prealloc bool
+
+	// SevPolicy is the policy for the SEV instance. For more info, see AMD document 55766
+	// This is only relevant for sev-guest objects
+	SevPolicy uint32
+
+	// SevCertFilePath is the path to the guest Diffie–Hellman key
+	// This is only relevant for sev-guest objects
+	SevCertFilePath string
+
+	// SevSessionFilePath is the path to the launch blog
+	// This is only relevant for sev-guest objects
+	SevSessionFilePath string
+
+	// SevKernelHashes specifies whether the hashes of the kernel, initrd, & cmdline are included in the measurement
+	// This is only relevant for sev-guest objects
+	SevKernelHashes bool
```

Comment:
This is another part that I think that could be made opaque.

Reply:
We could define a `SevGuestObject` type that inherits from `type Object struct`. Although, I'm not sure that would result in "better" code. Eh, I'm happy with how it is.

Reply:
Or simply name it in a way it can be used for both SEV and TDX, for instance.
But that's not for this PR, it was just a note for the future.